dcp.yml

```diff
@@ -0,0 +1,33 @@
+version: "3.7"
```

Comment:
There are to files at the moment: dcp.yml and docker-compose.yml they seem pretty redundant.